ddtrace/profiling/collector/_lock.py

```diff
@@ -64,13 +65,16 @@ class _ProfiledLock(wrapt.ObjectProxy):
     ACQUIRE_EVENT_CLASS = LockAcquireEvent
     RELEASE_EVENT_CLASS = LockReleaseEvent
 
-    def __init__(self, wrapped, recorder, tracer, max_nframes, capture_sampler, endpoint_collection_enabled):
+    def __init__(
+        self, wrapped, recorder, tracer, max_nframes, capture_sampler, endpoint_collection_enabled, export_libdd_enabled
```

Comment:
Follow-up: we should consider adding type hints here at some point

Reply:
Seems reasonable, will take a peek

Reply:
Added